kclvm/cmd/src/settings.rs

```diff
@@ -33,9 +33,29 @@ pub(crate) fn build_settings(matches: &ArgMatches) -> Result<SettingsPathBuf> {
         None => vec![],
     };
     let output = matches.value_of("output").map(|v| v.to_string());
+    let setting_files = if let Some(files) = matches.values_of("setting") {
+        Some(files.into_iter().collect::<Vec<&str>>())
+    } else {
+        None
+    };
+    _build_settings(
+        files,
+        output,
+        setting_files,
+        matches.occurrences_of("debug") > 0,
+        matches.occurrences_of("disable_none") > 0,
+    )
+}
 
+pub fn _build_settings(
```

Comment:
s/_build_settings/build_settings, and use `build_settings_with_matches` instead of `build_settings`, put `build_settings` into the config crate if the kcl language server needs.